src/app.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```

Comment:
Since this is a component, you need to move it into the component folder and get rid of the index.jsx in the component folder.